src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC, memo, useMemo } from 'react';
+import styled from 'styled-components';
+
+import { gray100 } from '../../core/Colors';
+
+export interface SkeletonProps {
+  style?: React.CSSProperties;
+  className?: string;
+  circle?: boolean;
+  rounded?: boolean;
+  width?: number | string;
+  height?: number | string;
+  count?: number;
+  color?: string;
+  lineHeight?: number;
+}
+
+export const Skeleton: FC<SkeletonProps> = memo(
+  ({ style, className, circle, width, height, count, color, lineHeight, rounded }) => {
+    const content = useMemo(() => {
+      let result = '';
+      for (let i = 0; i < (count || 1); i += 1) {
+        result += 'F';
+      }
+      return result;
+    }, [count]);
+
+    return (
+      <Container
+        style={style}
+        className={className}
+        circle={circle}
+        width={width}
+        height={height}
+        count={count}
+        color={color}
+        lineHeight={lineHeight}
+        rounded={rounded}
+      >
+        <span>{content}</span>
+      </Container>
+    );
+  }
+);
+
+Skeleton.defaultProps = {
+  rounded: true,
+  color: gray100,
+  lineHeight: 20,
+};
+
+type ContainerProps = Pick<
+  SkeletonProps,
+  'circle' | 'width' | 'height' | 'rounded' | 'count' | 'color' | 'lineHeight' | 'rounded'
+>;
+
+const Container = styled.div<ContainerProps>`
+  ${props => props.width && `width: ${typeof props.width === 'number' ? `${props.width}px` : props.width}`};
+  ${props => props.height && `height: ${typeof props.height === 'number' ? `${props.height}px` : props.height}`};
+  ${props => props.count && `display: inline-block`};
+  ${props => props.rounded && `border-radius: 3px`};
+  ${props => props.circle && `border-radius: 50%`};
+  line-height: ${props => `${props.lineHeight}px`};
+  background-color: ${props => props.color};
+  > span {
+    opacity: 0;
+  }
+  @keyframes fadeOut {
```

Comment:
```suggestion
// Create the keyframes
const fadeOut = keyframes`
    from {
      opacity: 1;
    }
    to {
      opacity: 0.25;
    }
`;
```
Container 바깥에서 선언할 수 있어요 
https://www.styled-components.com/docs/basics#animations